### PR TITLE
Remove unused code.

### PR DESCRIPTION
**Pull request by troessner**
Coming from #745

### CODE REVIEW

File: spec/reek/cli/options_spec.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@
   describe 'parse' do
     it 'raises on invalid argument in ARGV' do
       options = described_class.new
-      options.argv = ['-z']
+      options.instance_variable_set :@argv, ['-z']
```
Comment (mvz) [inline spec/reek/cli/options_spec.rb]: Can we solve this some other way? I can look into options later ...

Comment (troessner) [inline spec/reek/cli/options_spec.rb]: >> Can we solve this some other way? I can look into options later ...

You don't like instance_variable_set? The madness!

Comment (chastell) [inline spec/reek/cli/options_spec.rb]: Why can’t we just pass this to `described_class.new` above?

Comment (troessner) [inline spec/reek/cli/options_spec.rb]: Ah yeah OR that . :laughing: 
Updated!

**Comment by chastell** [comment]
:raised_hands:

**Pull request merged**